lib/plugins/attributeAlias.js

```diff
@@ -93,14 +93,11 @@ function applyAlias(mappings) {
  */
 function updateAttribute(entity, typeInformation, callback) {
     const mappings = extractAllMappings(typeInformation);
-    if (config.isCurrentNgsi()) {
```

Comment:
We are removing usages of `isCurrentNgsi()` in IOTA agent code (e.g. PR https://github.com/telefonicaid/iotagent-ul/pull/467 PR  https://github.com/telefonicaid/iotagent-json/pull/533) so I'd suggest to remove also the `isCurrentNgsi()`  function itself in the IOTA Lib.